### ISSUE TITLE
Support Request: Cant change area/ wouldnt use it

### ISSUE THREAD

**Issue by Dauerst0ned** [OpenTabletDriver/OpenTabletDriver#2518]
### Acknowledgements

- [X] I have installed OpenTabletDriver by following its [official installation instructions](https://opentabletdriver.net/).
- [X] I have checked the [OpenTabletDriver Wiki](https://opentabletdriver.net/Wiki) and respective FAQ pages, and my issue was not covered or actually fixed.
- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.

### What do you need help with?

After being gone for a month i want to play osu but my are isnt working.. i tried to change it. Also i reinstalled it and doenst work either. How can i fix it, what is the problem

### What have you tried?

Reinstalling, restart pc.

### Diagnostic Information

[diagnostic.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/10141926/diagnostic.txt)

**Comment by jamesbt365** [comment]
- Open "Device Manager"
- At the top click view > device by container
- Look at the list of devices
- Find your tablet
- Open the dropdown menu (container) next to the tablet
- In the drop down container, find any of the following and right click them: <OEM> tablet (e.g. “Wacom Tablet”; “Gaomon Tablet”), the model number (e.g. “CTL-472”), or the name of the tablet (“Wacom Intuos S”)
- Click on update -> browse my computer for drivers -> let me pick from a list of available drivers -> usb input device, usb composite device, or hid compliant device -> next
- Repeat the first two steps if any other matching devices are present
- Replug your tablet and check if otd detects it

Also there is considerable amount of duplicates

**Comment by Dauerst0ned** [comment]
where is device manager i cant find it

**Comment by jamesbt365** [comment]
an application on your computer.

**Comment by Dauerst0ned** [comment]
bruh

**Comment by Dauerst0ned** [comment]
maybe its also another problem because its still doesnt work
The thing is i have 2  monitors one is 24 and one 27 and if i use display 1 as area it still goes to display 2 a little bit

**Comment by jamesbt365** [comment]
Screenshot device manager with the container for your tablet open

**Comment by Dauerst0ned** [comment]
![image](https://user-images.githubusercontent.com/119682492/205328840-06fd5266-3abd-43c8-b064-ec0d8f34fd12.png)
im Germany so sry if you cant understand something

**Comment by Dauerst0ned** [comment]
there was also this error
![image](https://user-images.githubusercontent.com/119682492/205329024-809ab303-b9ac-4ade-a77e-1803a7bb7429.png)

**Comment by Dauerst0ned** [comment]
hey i checked the problem with my other tablet. and the problem is definitly not my tablet ... / driver

**Comment by jamesbt365** [comment]
right click hid complaint mouse > update > browse my computer > let me pick from a list of available drivers > screenshot.

**Comment by Dauerst0ned** [comment]
![image](https://user-images.githubusercontent.com/119682492/205377269-1454560b-0149-4d1e-9a50-314f56b04ca9.png)

**Comment by jamesbt365** [comment]
bottom one -> next. Once it is done replug your tablet.

**Comment by Dauerst0ned** [comment]
still doesnt work.. the thing is technically i have right area/ sens but it doesnt stop there where i limited it and it is postponed

**Comment by Dauerst0ned** [comment]
like if i go in the middle it is on right top

**Comment by Dauerst0ned** [comment]
so i tried it on another pc and it also doesnt work. So it should be problems with the tablet drivers. Right?

**Comment by jamesbt365** [comment]
No. Windows has been pushing broken device drivers for months now.

**Comment by jamesbt365** [comment]
Screenshot device manager again.

**Comment by Dauerst0ned** [comment]
![image](https://user-images.githubusercontent.com/119682492/205452886-9ed289dd-93d9-47ba-8cc3-0ebea3096809.png)

**Comment by Dauerst0ned** [comment]
![image](https://user-images.githubusercontent.com/119682492/205452913-c2ba535a-6488-4d53-9e6f-8bfd8bd50bc2.png)

**Comment by Dauerst0ned** [comment]
i first it maybe was win 11 
but on win 10  also doesnt work

**Comment by jamesbt365** [comment]
right click ctl-472 > update > browse my computer > let me pick from a list of available drivers > usb input/composite device > next > finish > replug.

**Comment by Dauerst0ned** [comment]
wanst the problem

**Comment by Dauerst0ned** [comment]
![image](https://user-images.githubusercontent.com/119682492/205453094-501900df-812e-402f-9cb0-0241e08984fb.png)
u see my are and if i go to the corner it goes on my second monitor

**Comment by jamesbt365** [comment]
Join the [Discord](https://discord.com/invite/9bcMaPkVAR) server. I'll deal with this there.

**Comment by Dauerst0ned** [comment]
i joined
